Lib/slapdtest/_slapdtest.py

```diff
@@ -407,12 +407,14 @@ def _start_slapd(self):
             '-F', self._slapd_conf,
             '-h', ' '.join(urls),
         ]
+        stderr = None
         if self._log.isEnabledFor(logging.DEBUG):
             slapd_args.extend(['-d', '-1'])
+            stderr = os.open(os.path.join(self.testrundir, 'slapd.log'), os.O_WRONLY|os.O_CREAT)
         else:
             slapd_args.extend(['-d', '0'])
         self._log.info('starting slapd: %r', ' '.join(slapd_args))
-        self._proc = subprocess.Popen(slapd_args)
+        self._proc = subprocess.Popen(slapd_args, stderr=stderr)
```

Comment:
I think the file descriptor opened in `stderr = os.open` is not closed anywhere.
Should we close it after Popen, or store it and close in _stopped() or wait().